### DISCUSSION TITLE
页面滚动条怎么统一去掉

### DISCUSSION THREAD

**Discussion by jiangzm** [NervJS/taro#10357] [Q & A]
目前遇到一个问题需要把小程序所有页面`Page`滚动条隐藏， 试了很多方案好像都不生效。

原生微信小程序设置下面的样式是可以生效，taro小程序不行。
```
::-webkit-scrollbar {
    width: 0;
    height: 0;
    color: transparent;
}
```

**Comment by Chen-jj** [comment]
编译后的样式代码还存在这个样式吗？

**Reply by jiangzm** [reply]
存在的

**Comment by vdfor** [comment]
::-webkit-scrollbar {
     display: none;
     width: 0 !important;
     height: 0 !important;
     background: transparent;
}
